src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/DebugInfoTableNode.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+using Internal.JitInterface;
+using Internal.NativeFormat;
+using Internal.Text;
+using Internal.TypeSystem;
+
+namespace ILCompiler.DependencyAnalysis.ReadyToRun
+{
+    public class DebugInfoTableNode : HeaderTableNode
```

Comment:
The code in this class could benefit from extra documentation.

Reply:
I added an overview of what information we put into the image.